### ISSUE TITLE
[MAUI] TabHostView styling causes NullReferenceException

### ISSUE THREAD

**Issue by janusw** [roubachof/Sharpnado.Tabs#138]
**Platform (please complete the following information):**
 - OS: Android
 - Device: Samsung Galaxy S21
 - Sdk vervion: Android SDK 34
 - MAUI: 8.0.82

**Describe the bug**
When trying to style a `TabHostView` via `Styles.xaml`, I see a `NullReferenceException` when the `TabHostView` is created.

**To Reproduce**
1. Download the sample project from https://github.com/janusw/Sharpnado.Tabs/tree/shellsample_styling/MauiShellSample
2. Build and run it.
3. Click the button ("Click me").
4. App crashes with exception.

**Exceptions (if applicable)**
```
[mono-rt] [ERROR] FATAL UNHANDLED EXCEPTION: System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation.
[mono-rt]  ---> System.NullReferenceException: Object reference not set to an instance of an object.
[mono-rt]    at Sharpnado.Tabs.TabHostView.UpdateTabType() in Sharpnado.Tabs/Tabs/Tabs/TabHostView.cs:line 594
[mono-rt]    at Sharpnado.Tabs.TabHostView.OnPropertyChanged(String propertyName) in Sharpnado.Tabs/Tabs/Tabs/TabHostView.cs:line 314
```

**Comment by roubachof** [comment]
fixed by v3.2.1

**Comment by janusw** [comment]
> fixed by v3.2.1

Wow, that was quick! 😃 

And yes, I can confirm that 3.2.1 works well in that regard. Thanks for fixing!
